DMX.Core.Api.Tests.Unit/Usings.cs

```diff
@@ -0,0 +1 @@
+global using Xunit;
```

Comment:
Visual Studio automatically added this. I suspect it could be replaced by add the Xunit using to each test class instead.

Reply:
@mimendel delete this whole file.
Then go to the .csproj file and disable global usings and null checks.

here's an example:
```xml
        <PropertyGroup>
		<TargetFramework>net7.0</TargetFramework>
		<Nullable>disable</Nullable>
		<ImplicitUsings>disable</ImplicitUsings>
	</PropertyGroup>

```